projects/storefrontstyles/scss/layout/page-templates/_order-detail-page.scss

```diff
@@ -1,5 +1,117 @@
-%AccountPageTemplate-calydon {
+// %AccountPageTemplate-calydon {
+//   cx-order-details-headline {
+// @include media-breakpoint-down(sm) {
+//   border: none;
+// }
+
+// .cx-header {
+//   border: 1px solid;
+//   border-color: var(--cx-color-light);
+//   padding: 30px 30px 20px 30px;
+//   margin-bottom: 30px;
+//   margin-left: 0;
+//   margin-right: 0;
+
+//   @include media-breakpoint-down(sm) {
+//     border: none;
+//     padding: 0;
+//   }
+
+//   .cx-detail {
+//     border-right: 1px solid var(--cx-color-light);
+
+//     &:last-child {
+//       border: none;
+//     }
+
+//     @include media-breakpoint-up(md) {
+//       padding-left: 47px;
+
+//       &:first-child {
+//         padding-left: 0;
+//       }
+//     }
+
+//     @include media-breakpoint-down(sm) {
+//       border: none;
+//       margin-bottom: 10px;
+//     }
+//   }
+
+//   .cx-detail-label {
+//     font-size: 18px;
+//     font-weight: bold;
+//   }
+// }
+// }
+
+// cx-order-details-items {
+// margin-bottom: 15px;
+// .cx-list-header {
+//   @include media-breakpoint-down(sm) {
+//     padding: 0;
+//   }
+// }
+
+// .cx-list {
+//   @include media-breakpoint-down(sm) {
+//     margin-right: 0;
+//     margin-left: 0;
+//   }
+// }
+
+// .cx-list-status {
+//   padding: 15px 30px;
+//   background-color: var(--cx-color-background);
+//   @include type('4');
+//   font-weight: bold;
+// }
+
+// .cx-list-date {
+//   margin: 0.5rem 0 0;
+// }
+// }
+
+// cx-order-details-shipping {
+// .cx-summary {
+//   margin-bottom: 48px;
+// }
+
+// .cx-account-summary {
+//   background-color: var(--cx-color-background);
+
+//   @include media-breakpoint-up(md) {
+//     padding: 5px 18px 12px;
+//   }
+
+//   @include media-breakpoint-down(sm) {
+//     margin: 0 -24px;
+//   }
+
+//   .cx-card-title {
+//     @include type('4');
+//     font-weight: bold;
+//   }
+
+//   .cx-card-label-bold {
+//     font-weight: bold;
+//   }
+// }
+// }
+
+//   cx-order-details-totals {
+//     margin-bottom: 32px;
+//   }
+// }
+
+%AccountPageTemplate {
+  --cx-max-width: 100%;
+  @extend %LoginPageTemplate !optional;
+  @extend %LoginPageTemplate-#{$theme} !optional;
+
   cx-order-details-headline {
+    order: 1;
```

Comment:
I hope we can drop this explicit `order`. why do we have it?

Reply:
I'm all for it  but (it breaks keyboard accessibility)... we have it because it was introduced about a year ago by developers, we need to have a closer look at this one, i'll create a specific ticket for this, since other slots are also being reorder in this same file... **no changes here for now**.